test/unit/models/tasks.test.js

```diff
@@ -91,6 +91,14 @@ describe("tasks", function () {
         expect(task).to.contain.all.keys(sameTask);
       });
     });
+    it("should fetch tasks filtered by search term", async function () {
+      const searchTerm = "task";
+      const tasksSnapshot = await tasksModel.get();
+      const result = await getBuiltTasks(tasksSnapshot, searchTerm);
```

Comment:
can we check the length here as well ! to be sure that its not an empty array and the below assertions are working.

Reply:
done